spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -126,28 +135,28 @@
 	 * <li>deployment status computation to the provided {@link AppDeployer}</li>
 	 * </ul>
 	 *
-	 * @param repository the repository this controller will use for stream CRUD
-	 * operations
-	 * @param deploymentIdRepository the repository this controller will use for
-	 * deployment IDs
-	 * @param deploymentController the deployment controller to delegate deployment
-	 * operations
+	 * @param repository the repository this controller will use for stream CRUD operations
+	 * @param deploymentIdRepository the repository this controller will use for deployment
+	 * IDs
+	 * @param deploymentController the deployment controller to delegate deployment operations
 	 * @param deployer the deployer this controller will use to compute deployment status
 	 * @param appRegistry the app registry to look up registered apps
 	 */
 	public StreamDefinitionController(StreamDefinitionRepository repository,
 			DeploymentIdRepository deploymentIdRepository, StreamDeploymentController deploymentController,
-			AppDeployer deployer, AppRegistry appRegistry) {
+			AppDeployer deployer, AppRegistry appRegistry, DefaultStreamService defaultStreamService) {
```

Comment:
Shouldn't we use its interface `StreamService`?

Reply:
yep, intellij refactoring cruft.